### ISSUE TITLE
do you support python 3.12

### ISSUE THREAD

**Issue by dbspace** [databricks/databricks-sql-python#429]
In your document https://docs.databricks.com/en/dev-tools/sqlalchemy.html it mention we need python <=3.11, but I also see a few close issue related to 3.12, so are you officially support 3.12

**Comment by jprakash-db** [comment]
@dbspace Yes, we will update the documentation. But it is safe to use in python 3.12
